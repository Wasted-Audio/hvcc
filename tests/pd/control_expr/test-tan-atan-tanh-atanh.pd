#N canvas 1119 295 450 300 12;
#X obj 52 25 loadbang;
#X obj 52 260 print;
#X obj 52 100 t b b b;
#X msg 161 135 5.5;
#X msg 52 135 2.3;
#X obj 52 58 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 168 260 print;
#X obj 290 260 print;
#X msg 106 135 -3.1;
#X obj 52 227 expr tan($f1);
#X obj 168 227 expr atan($f1);
#X obj 290 227 expr tanh($f1);
#X obj 426 228 expr atanh($f1);
#X obj 426 254 print;
#X connect 0 0 5 0;
#X connect 2 0 4 0;
#X connect 2 1 8 0;
#X connect 2 2 3 0;
#X connect 3 0 9 0;
#X connect 3 0 10 0;
#X connect 3 0 11 0;
#X connect 3 0 12 0;
#X connect 4 0 9 0;
#X connect 4 0 10 0;
#X connect 4 0 11 0;
#X connect 4 0 12 0;
#X connect 5 0 2 0;
#X connect 8 0 9 0;
#X connect 8 0 10 0;
#X connect 8 0 11 0;
#X connect 8 0 12 0;
#X connect 9 0 1 0;
#X connect 10 0 6 0;
#X connect 11 0 7 0;
#X connect 12 0 13 0;
