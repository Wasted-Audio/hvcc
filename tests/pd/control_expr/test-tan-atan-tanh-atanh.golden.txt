[@ 0.000] print: -0.995584
[@ 0.000] print: 1.39094
[@ 0.000] print: 0.999967
[@ 0.000] print: -nan
[@ 0.000] print: 0.0416167
[@ 0.000] print: -1.25875
[@ 0.000] print: -0.995949
[@ 0.000] print: -nan
[@ 0.000] print: -1.11921
[@ 0.000] print: 1.16067
[@ 0.000] print: 0.980096
[@ 0.000] print: -nan
